### PR TITLE
Add suggestion about updating pip in the user site-packages

### PR DESCRIPTION
**Pull request by Binary-Eater**
Provide a suggestion for a non-intrusive update of pip. Updating pip may resolve issues where some previous versions would fail to properly validate version compatibility for dependencies that were previously installed through the system package manager.

### CODE REVIEW

**Comment by Binary-Eater** [comment]
Figured it would be good to document this until formal packaging is part of the release.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ cd git-credential-msal
 python3 -m pip install .
 ```
 
+If you run into issues with the above, first verify that the above works with
```
Comment (aaronp24) [inline README.md]: "the above" sounds a little weird twice in this sentence. Maybe "If you run into issues with the above, verify that the latest version of `pip` is installed." ?

File: README.md [hunk 2]
```diff
@@ -23,6 +23,15 @@ cd git-credential-msal
...
+the latest version of `pip`. To avoid an conflicts with a system's package
```
Comment (aaronp24) [inline README.md]: Maybe "To avoid a conflict with the system's package manager, ..."?

**Comment by Binary-Eater** [comment]
Thanks @aaronp24. Remind me not to write documentation before going to bed........

**Review by aaronp24 — APPROVED** [review]

**Pull request merged**